src/main/markdown/usingeclipse.md

```diff
@@ -15,21 +15,21 @@ GWT even easier.
 
 ## Download Eclipse<a id="eclipse"></a>
 
-If you do not already have Eclipse, you may download it from the [Eclipse Website](http://www.eclipse.org/downloads/). We suggest downloading Eclipse 4.3 (Kepler).
+If you do not already have Eclipse, you may download it from the [Eclipse Website](http://www.eclipse.org/downloads/). We suggest downloading Eclipse 4.4 (Luna).
 
 ## Install the Plugin<a id="installing"></a>
 
 <img src="https://developers.google.com/eclipse/images/google-plugin.png" style="float: left; width: 40px;
 margin-right: 15px;"/>
 
-Install the Google Plugin for Eclipse 4.3 by using the following update site:
+Install the Google Plugin for Eclipse 4.4 by using the following update site:
```

Comment:
I think that would be better, this way we do not have to keep track of all GPE releases